test/thundercat-router-test.js

```diff
@@ -0,0 +1,126 @@
+'use strict';
```

Comment:
**TODO for lost points**

Since Duncan said no need for a GET 400 test, the only tests you're missing are GET 200 and 404.